src/ImageBackingStore.cpp

```diff
@@ -1,8 +1,8 @@
 /**
- * ZuluSCSI™ - Copyright (c) 2022 Rabbit Hole Computing™
+ * Portions - Copyright (C) 2023 Eric Helgeson
+ * ZuluSCSI™ - Copyright (c) 2022-2023 Rabbit Hole Computing™
```

Comment:
Order of copyright notices seems wrong, most of the code in this file is from Rabbit Hole Computing.